eclass/gkrellm-plugin.eclass

```diff
@@ -51,11 +58,14 @@ esac
 
 inherit multilib
 
-EXPORT_FUNCTIONS src_install
-
 if [[ ! ${_GKRELLM_PLUGIN_R1} ]]; then
+_GKRELLM_PLUGIN_R1=1
 
-DEPEND="virtual/pkgconfig"
+if [[ ${EAPI} == 6 ]]; then
+	DEPEND="virtual/pkgconfig"
+else
+	BDEPEND="virtual/pkgconfig"
+fi
 
 # @FUNCTION: gkrellm-plugin_src_install
 # @DESCRIPTION:
```

Comment:
According to the [eclass writing guide](https://devmanual.gentoo.org/eclass-writing/index.html#eclass-functions) the `@USAGE:` is required, but I've spotted a few other eclasses where `@USAGE:` is missing (I think it's the EXPORTED_FUNCTIONS).

Should we add it?

Reply:
just add it, yes

Reply:
done